src/reducers/postEdits.js

```diff
@@ -122,7 +125,13 @@ export const tagDescriptionEdits = (state = {}, action) => {
   } else if (includes([START_POST_EDIT, CANCEL_POST_EDIT, CREATE_COMMENT], type)) {
     return {}
   } else if (type === EDIT_TAG_DESCRIPTION) {
-    return {...state, [payload.tag]: payload.description}
+    if (state.creating === true) {
```

Comment:
no, because if `creating === true`, then payload.tag keeps changing, so if you merge `{[payload.tag]: payload.description}` with previous state, you end up with
```
{
  t: 'description',
  ta: 'description',
  tag: 'description',
  tagn: 'description',
  tagna: 'description'
}
this is also why I elected to store creating: true in this reducer rather than giving it it's own, because this reducer needs to know which mode we're in.
I'll think about if there's another way to do it, and if you see one please let me know, I'd **love** to not have this `creating === true` stuff.